ydb/docs/en/core/changelog-server.md

```diff
@@ -1,5 +1,85 @@
+* [Optimized](https://github.com/ydb-platform/ydb/pull/9491) the header size of large binary objects in VDisk.
+* Improved diagnostics and introspection of memory errors.
+* Reduced memory consumption through allocator page cleaning.
+
+#### Bug Fixes
+
+* [Fixed](https://github.com/ydb-platform/ydb/pull/9707) an error in [Interconnect](../concepts/glossary#actor-system-interconnect) configuration that led to performance degradation.
+* [Fixed](https://github.com/ydb-platform/ydb/pull/13993) the out of memory error when deleting very large tables by regulating the number of tablets simultaneously processing this operation.
+* [Fixed](https://github.com/ydb-platform/ydb/pull/9848) an issue with handling accidental duplicate nodes in the system tablet configuration.
+* [Fixed](https://github.com/ydb-platform/ydb/pull/11059) an issue where reading data would take too long (seconds) during frequent table resharding operations.
+* [Fixed](https://github.com/ydb-platform/ydb/pull/9723) an error in reading from asynchronous replicas that led to failure.
+* [Fixed](https://github.com/ydb-platform/ydb/pull/9507) an issue with rare [CDC](./dev/cdc.md) initial scan freezes.
+* [Fixed](https://github.com/ydb-platform/ydb/pull/11483) an issue with handling incomplete schema transactions in datashards during system restart.
+* [Fixed](https://github.com/ydb-platform/ydb/pull/10460) an issue with inconsistent reading from a topic when trying to explicitly confirm a message read within a transaction. Now the user will receive an error when attempting to confirm a message.
+* [Fixed](https://github.com/ydb-platform/ydb/pull/12220) an issue where topic auto-partitioning worked incorrectly within a transaction.
+* [Fixed](https://github.com/ydb-platform/ydb/pull/12905) an issue where a transaction hangs when working with topics during tablet restarts.
+* [Fixed](https://github.com/ydb-platform/ydb/pull/13910) the "Key is out of range" error when importing data from S3-compatible storage.
+* [Fixed](https://github.com/ydb-platform/ydb/pull/13741) an issue where the end of the metadata field in the [dynamic cluster configuration](../maintenance/manual/dynamic-config) was determined incorrectly.
+* [Improved](https://github.com/ydb-platform/ydb/pull/16420) the process of building secondary indices: when certain errors occur, the system retries the process instead of interrupting it.
+* [Fixed](https://github.com/ydb-platform/ydb/pull/16635) an error in executing the `RETURNING` expression in INSERT/UPSERT operations.
+* [Fixed](https://github.com/ydb-platform/ydb/pull/16269) an issue with Drop Tablet operations in the PQ tablet hanging, especially during Interconnect delays.
+* [Fixed](https://github.com/ydb-platform/ydb/pull/16194) an error that occurred during VDisk [compaction](../concepts/glossary#compaction).
+* [Fixed](https://github.com/ydb-platform/ydb/pull/15233) an issue where long topic reading sessions ended with "too big inflight" errors.
```

Comment:
bettter quote a more complete error message in backticks so this has better chances to be found by the error message